packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.17.1] - 2021-06-29
+### Fixed
+- Fix query greater than problem, query with max range to speed up dictionary (#365)
```

Comment:
```suggestion
- Fix an edge case for dictionary query, add blocknumber max range to speed up dictionary (#365)
```

Reply:
we can add a priority field to instruct how urgent this release is.
possible values: low, medium, high